assessments/0003-krator.md

```diff
@@ -0,0 +1,181 @@
+# Krator Docs assessment
+
+## Introduction
+
+
+This is an assessment of your project documentation and website (if present), prepared for you and your project by the CNCF techdocs team.
+
+This document aims to:
+
+- Measure existing documentation quality against the CNCF’s standards
+- Recommend specific and general improvements
+- Provide examples of great documentation as reference 
+- Identify key areas which will net the largest improvement if addressed 
+
+Since the Krator documentation site is just getting started, we'll have to create or significantly develop the current content. This assessment provides a starting point for articulating current state and ideas for the future.
+
+## How this document works
+
+The assessment is divided into three sections: 
+
+- **Project documentation:** for end users of the project; aimed at people who intend to use it
+- **Contributor documentation:** for new and existing contributors to the project
+- **Website:** branding, website structure, and maintainability
+
+Within each section you receive a rating on different criteria. The full definition of each criteria is defined in [the criteria](criteria.md). We recommend reading each criterion’s definition.
+
+## Project documentation 
+
+
+| Criteria                   | 1    | 2   | 3   | 4   | 5   |
+| ---                        | ---  | --- | --- | --- | --- |
+| Information architecture   |  ✅  |     |     |     |     |
+| New user content           |  ✅  |     |     |     |     |
+| Content maintainability    |  ✅  |     |     |     |     |
+| Content creation processes |  ✅  |     |     |     |     |
+
+ Criteria: 
+ - 1 = (Is not present or requires significant work) 
+ - 3 = (is present, but needs work) 
+ - 5 = (is executed extremely well or no improvement required)
+
+### Comments
+
+- **Information Architecture**: 
+    - Documentation is currently in several locations and will need to be brought into one repo. The current resources are:
+        - The project [README](https://github.com/krator-rs/krator)
+        - [Introduction to krator Blog Post](https://deislabs.io/posts/introducing-krator/)
+        - [A Fistful of States: More State Machine Patterns in Rust](https://deislabs.io/posts/a-fistful-of-states/)
+        - [Crate krator API Documentation](https://docs.rs/krator/0.4.0/krator/). This is a substantial part of the current documentation and could be incorporated into the main body (site/repo) of the documentation. This is already a great resource for users, but if we transport it to a documentation repo, it will take some work if it's not already in markdown. Is there a way extract/download these guides quickly?
```

Comment:
One thing to note about their API documentation is that it is auto-generated as a part of the Rust Crate. While this offers convenience for the developers, it makes it difficult for non-developer users to contribute to the documentation. One thing to talk about with the devs is if they're okay with this tradeoff.